lib/matplotlib/mathtext.py

```diff
@@ -825,13 +834,20 @@ def _get_glyph(self, fontname, font_class, sym, fontsize, math=True):
                     warnings.warn(
                         "Substituting with a symbol from Computer Modern.",
                         MathTextWarning)
+                elif isinstance(self.cm_fallback, StixFonts):
+                    warnings.warn(
+                        "Substituting with a symbol from STIX.",
+                        MathTextWarning)
                 if (fontname in ('it', 'regular') and
                         isinstance(self.cm_fallback, StixFonts)):
                     return self.cm_fallback._get_glyph(
```

Comment:
I think you could just set `fontname='rm'` in this line, get rid of the else block below, and then just have a single `return self.cm_fallback._get_glyph(fontname, font_class, sym, fontsize)` instead of three.

Reply:
Thanks for pointing this out. I fixed this a couple of days ago.